packages/elements-react/src/theme/default/components/card/current-identifier-button.tsx

```diff
@@ -41,6 +43,7 @@ export function DefaultCurrentIdentifierButton() {
       {...attributes}
       href={initFlowUrl}
       title={`Adjust ${nodeBackButton?.attributes.value}`}
+      data-testid={"ory/ui/login/link/restart"}
```

Comment:
We now have very different looking test IDs. I'd like to see this change be done for all test IDs at once. Or as followup, but I fear that it will be forgotten then...

Reply:
Agree, but I'm not going to refactor all test IDs in this PR

Reply:
I guess I can do it in a followup then...

Reply:
We anyways have an absolute mess of test-ids. We have different test ids in elements v1, ax v0, no-elements react native, and elements v2. in the cloud e2e test pr i have resolved those issues with a clever workaround. I really don't think it's necessary to refactor all of this at the moment because it's working as far as i can see

Reply:
>We anyways have an absolute mess of test-ids.

But this makes it worse - I don't care about the particular format, just trying to not make it even messier in a library that has few consumers yet. 

I'll do it in a followup. From my POV, this PR is good to go now...

Reply:
Maybe we add a flag to exclude test IDs unless we build it with ax, so that they're not exposed to end-users